app/views/posts/show.html.erb

```diff
@@ -5,7 +5,27 @@
     <div class="h-full flex items-center border-gray-200 border p-4 rounded-lg">
       <div class="flex-grow">
         <h2 class="text-gray-900 title-font font-medium">Post #<%= @post_index + 1 %> by <%= author.name %> </h2>
```

Comment:
- [ ] Also the line `7` make sure we are displaying the `title` created, let's take an example from this:

`<h2 class="text-gray-900 title-font font-medium"><%=@post.title %> by <%= author.name %> </h2>`